addons/website/static/tests/tours/snippet_empty_parent_autoremove.js

```diff
@@ -6,6 +6,9 @@ function removeSelectedBlock() {
     return {
         content: "Remove selected block",
         trigger: '#oe_snippets we-customizeblock-options:nth-last-child(3) .oe_snippet_remove',
+        run() {
+            this.anchor.click();
+        }
```

Comment:
Can't this just be removed? Isn't click default behavior? It has to be explicitly marked as click now, isn't it? But is this way the shortest ? Seems verbose for just a click
`run: "click"` ?

Reply:
Done